cpp/kiss_icp/pipeline/KissICP.hpp

```diff
@@ -57,10 +57,7 @@ class KissICP {
           local_map_(config.voxel_size, config.max_range, config.max_points_per_voxel),
           adaptive_threshold_(config.initial_threshold, config.min_motion_th, config.max_range) {}
 
-    KissICP()
-        : local_map_(config_.voxel_size, config_.max_range, config_.max_points_per_voxel),
-          adaptive_threshold_(config_.initial_threshold, config_.min_motion_th, config_.max_range) {
-    }
+    KissICP() : KissICP(KISSConfig{}) {}
```

Comment:
Also not strictly necessary to fix the warnings, but seemed like a good chance to reduce redundancy and more clearly state that we're using the `KISSConfig` defaults.

Reply:
Yes! amazing! we love to chope down code